servicenow/servicenow.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * Copyright 2020 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+(function process(/*RESTAPIRequest*/ request, /*RESTAPIResponse*/ response) {        
+
+var writer = response.getStreamWriter();
+hdrs = {};
+var requestData=request.body.data;
+var state='';
+var assigned_to='';
+var context='';
+var message='';
+data={};
+
+//create a new ticket
+  if(requestData.queryResult.intent.displayName=='incident.create.details') {                                
```

Comment:
For rest part of this function, they don't have indent. You could just remove the indent for this part as well.

Reply:
I'm not following.  I thought I was to add more indents.

Reply:
all indents line up from beginning to end